primitive/src/main/java/io/atomix/primitive/session/impl/AbstractSession.java

```diff
@@ -47,7 +47,7 @@ protected AbstractSession(
     this.sessionId = checkNotNull(sessionId);
     this.primitiveName = checkNotNull(primitiveName);
     this.primitiveType = checkNotNull(primitiveType);
-    this.memberId = checkNotNull(memberId);
+    this.memberId = memberId;
```

Comment:
was this an accident?

Reply:
No it's intentional. There's no member information with distributed log entries, so the sessions just set it to `null`